src/yang/ietf-bgp-common-multiprotocol.yang

```diff
@@ -86,8 +86,8 @@ submodule ietf-bgp-common-multiprotocol {
       type boolean;
       default "false";
       description
-        "This leaf indicates whether the IPv4 Unicast AFI,SAFI is
-         enabled for the neighbor or group";
+	"This leaf indicates whether this AFI,SAFI is enabled for
+	 the neighbor or group";
```

Comment:
Can you check for indentation (2 spaces), as it is hard to judge if the original code had a tab which is now replaced with spaces?

Also as you editing description statements, do you want to terminate them with a period.

Reply:
The indentation is fine.

See punctuation commentary elsewhere.